src/System.Security.Cryptography.Xml/tests/Samples/SigningVerifyingX509Cert.cs

```diff
@@ -60,6 +60,13 @@ private static bool VerifyXml(string signedXmlText, X509Certificate2 certificate
         }
 
         [Fact]
+        [SkipOnTargetFramework(
+            TargetFrameworkMonikers.Net45 |
+            TargetFrameworkMonikers.Net451 |
+            TargetFrameworkMonikers.Net452 |
+            TargetFrameworkMonikers.Net46 |
```

Comment:
@bartonjs if this is conceptually correct I'd leave this as is since we never know when we will need to run this test on lower versions